qiskit_nature/algorithms/ground_state_solvers/ground_state_eigensolver.py

```diff
@@ -87,11 +87,21 @@ def solve(
             An interpreted :class:`~.EigenstateResult`. For more information see also
             :meth:`~.BaseProblem.interpret`.
         """
+        main_operator, aux_ops = self.get_qubit_operators(problem, aux_operators)
+        raw_mes_result = self._solver.compute_minimum_eigenvalue(main_operator, aux_ops)  # type: ignore
+
+        result = problem.interpret(raw_mes_result)
+        return result
+
+    def get_qubit_operators(
+        self,
+        problem: BaseProblem,
+        aux_operators: Optional[ListOrDictType[Union[SecondQuantizedOp, PauliSumOp]]] = None,
+    ) -> Tuple[PauliSumOp, Optional[ListOrDictType[Union[SecondQuantizedOp, PauliSumOp]]]]:
```

Comment:
```suggestion
    ) -> Tuple[PauliSumOp, Optional[ListOrDictType[PauliSumOp]]]:
```
At the end of function, you should only have `PauliSumOp`s left.